CHANGELOG.md

```diff
@@ -1,3 +1,106 @@
+# 59.0.0
+
+This major release removed the remaining encoding/decoding behaviors from components ([tracking project](https://github.com/material-components/material-components-ios/projects/22)) and fixed a variety of bugs in FlexibleHeader with relation to safe area insets.
```

Comment:
Super-nit: The hyperlink should be around the descriptive text. That way a screen reader jumping between links can easily determine the entire context of the link.

Suggestion,
```
This major release [removed the remaining encoding/decoding behaviors from components (tracking project](https://github.com/material-components/material-components-ios/projects/22)) and fixed a variety of bugs in FlexibleHeader with relation to safe area insets.
```

Reply:
Done.